diff -r 1df7a6602edd trusty/debian/control
--- a/trusty/debian/control	Tue Jan 26 01:04:54 2016 +0100
+++ b/trusty/debian/control	Thu Jan 28 14:14:44 2016 -0800
@@ -41,7 +41,7 @@
      libusb-1.0-0-dev,
      libspnav-dev,
      libgdal-dev,
-     libsdformat4-dev,
+     libsdformat4-dev (>= 4.99.99+mentor2v1.0),
      libsimbody-dev (>= 3.0.0),
      libsimbody-dev (<< 4.0.0),
      libbullet2.82-dev,
@@ -148,7 +148,7 @@
 	 libavformat-dev,
 	 libavcodec-dev,
 	 libswscale-dev,
-         libsdformat4-dev,
+         libsdformat4-dev (>= 4.99.99+mentor2v1.0),
          libsimbody-dev (>= 3.0.0),
          libsimbody-dev (<< 4.0.0),
          libbullet2.82-dev,
