diff --git a/cmake/gazebo-config.cmake.in b/cmake/gazebo-config.cmake.in
--- a/cmake/gazebo-config.cmake.in
+++ b/cmake/gazebo-config.cmake.in
@@ -11,12 +11,6 @@
 # materials scripts, shaders, and other related resources are stored.
 set(@PKG_NAME@_MEDIA_PATH "@CMAKE_INSTALL_PREFIX@/share/gazebo-@GAZEBO_MAJOR_VERSION@/media")
 
-set(@PKG_NAME@_PLUGIN_PATH "@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_LIBDIR@/gazebo-@GAZEBO_VERSION@/plugins")
-
-# The media path contains the location on disk where images,
-# materials scripts, shaders, and other related resources are stored.
-set(@PKG_NAME@_MEDIA_PATH "@CMAKE_INSTALL_PREFIX@/share/gazebo-@GAZEBO_VERSION@/media")
-
 #################################################
 # @PKG_NAME@_PROTO_PATH, @PKG_NAME@_PROTO_INCLUDE_DIRS, and
 # @PKG_NAME@_PROTO_LIBRARIES

